subt/docker/robotika/Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN /usr/bin/python3 -m pip install --user --no-cache-dir \
   "opencv-python==4.2.0.32" \
   "pyzmq==19.0.0"
 
+RUN cd /home/developer/ && head -n -2 .bashrc > .bashrc.new && mv .bashrc.new .bashrc
```

Comment:
I suppose Dockerfile support comments and this line would surely deserve it

Reply:
It removes last 2 lines from `.bashrc` which contains sourcing of the original install space (from base image).

Reply:
yes, but ... this will fail when base image changes, right? I overlooked minus for -2, i.e. I have some idea what it does but no idea what is expected on the last two lines of base image

Reply:
I tried to say that it contains sourcing of the original install space.

Reply:
http://wiki.ros.org/catkin/workspaces#Development_.28Devel.29_Space